### PR TITLE
Change connection url

### PR DESCRIPTION
**Pull request by aherreDev**

### CODE REVIEW

File: src/services/totalsService.js [hunk 1]
```diff
@@ -22,12 +22,11 @@ class TotalsService extends EventTargetShim {
 
     startDate = startDate ? format(startDate, 'yyyy-MM-dd') : format(addDays(new Date(), -14), 'yyyy-MM-dd');
     endDate = endDate ? format(endDate, 'yyyy-MM-dd') : format(addDays(new Date(), 56), 'yyyy-MM-dd');
-
     const cacheKey = `${startDate}${endDate}`;
-
+    this.lastCacheKey = cacheKey;
```
Comment (KupVadim) [inline src/services/totalsService.js:26]: Are we using lastCacheKey somewhere?

File: src/services/totalsService.js [hunk 2]
```diff
@@ -22,12 +22,11 @@ class TotalsService extends EventTargetShim {
...
     if (opts.forceRefresh || this.cache[cacheKey]?.status === 'failed' || !this.cache[cacheKey]) {
       try {
-        this.cache[cacheKey] = {};
+        // this.cache[cacheKey] = {};
```
Comment (KupVadim) [inline src/services/totalsService.js:29]: I think that we need to reset the object in cache before making request

File: src/services/totalsService.js (hunk 1 above)
Comment (aherreDev) [reply src/services/totalsService.js:26]: Noup, I forget delete that line

File: src/services/totalsService.js (hunk 2 above)
Comment (aherreDev) [reply src/services/totalsService.js:29]: It's a singleton, if we delete the cache we drop the current logic, I think

Comment (KupVadim) [reply src/services/totalsService.js:29]: this.cache[cacheKey] contains result from API. But yeah, I see that it's not so important. In any way cache value will have result (result from API or in exception handler).

File: src/services/totalsService.js (hunk 1 above)
Comment (aherreDev) [reply src/services/totalsService.js]: It's done

**Pull request merged**